components/infobox/wikis/wildrift/infobox_item_custom.lua

```diff
@@ -236,7 +237,7 @@ function CustomInjector:parse(id, widgets)
 		if not (String.isEmpty(_args.sr) and
```

Comment:
this if checks for the existence of params that are not used in it, while it doesn't check for the existence of some of the params that are used